brayns/parameters/ParametersManager.cpp

```diff
@@ -127,7 +154,30 @@ VolumeParameters& ParametersManager::getVolumeParameters()
 
 void ParametersManager::set(const std::string& key, const std::string& value)
 {
+    const std::string p = "--" + key;
+    std::vector<std::string> strs;
+    boost::split(strs, value, boost::is_any_of(" "));
+
+    const size_t argc = 2 + strs.size();
+    const char** argv = new const char*[argc];
+    argv[0] = "";
+    argv[1] = p.c_str();
+    for (size_t i = 0; i < strs.size(); ++i)
+        argv[2 + i] = strs[i].c_str();
+
+    po::variables_map vm;
+    po::parsed_options parsedOptions =
+        po::command_line_parser(argc, argv)
+            .options(_parameters)
+            .style(po::command_line_style::unix_style ^
+                   po::command_line_style::allow_short)
+            .allow_unregistered()
+            .run();
+    po::store(parsedOptions, vm);
+    po::notify(vm);
+    delete[] argv;
+
     for (AbstractParameters* parameters : _parameterSets)
-        parameters->set(key, value);
+        parameters->parse(vm);
```

Comment:
I see this code has been moved as-is from AbstractParameters, but those 20 lines are still almost a copy-paste from the function above. Please try to reuse some code inside a common function.

Note: you also forgot to delete the AbstractParameters::set() declaration in the header file.